### ISSUE TITLE
Generate standalone scripts?

### ISSUE THREAD

**Issue by tin2tin** [snuq/Script-Shortcut#1]
Maybe there is no need for this, but I just came across this way to run scripts in the text editor from a script: https://blender.stackexchange.com/questions/8435/how-to-programmatically-load-a-python-script-in-text-editor-and-launch-it/31398#31398
And that made me think that if you at some point felt the need to export the generated buttons into a standalone script, it is actually possible to run the generated script automatically. :-)

Another thing is that Pablo has made a chat-room about the VSE and I've mentioned you vseqf as a potential add-on to include in Blender at some point. If you like, please join the discussion here: https://blender.chat/channel/vse

**Comment by snuq** [comment]
I'm not entirely understanding what you are suggesting to accomplish with this code?  Currently, Script Shortcut doesn't actually load any text into blender's text editor.  The method I chose does not need to use an operator (which I try to avoid), and it should properly handle dependencies and multi-file scripts as well (tho, I have not tested this).

**Comment by tin2tin** [comment]
No worries. You add-on is not a UI designer, and should not bleed into one. Is is very ingenious as it is. (One thing I was a bit puzzled about was how to actually use the buttons, until I realised I had to uncheck the editing box. Maybe this should somehow be made more clear. Up until that point I actually thought it was a kind of UI designer, leading up to my suggestion.) 

Nice to see the Windowless fullscreen operator here. I hope it eventually will make it into VSEQF. :-) 

On the other thing. If you at all feel like going in that direction, Pablo, wrote that maybe a VSE script could be included in 2.8. And I suggested yours, but also mentioned a need for a tools region on the left side of the sequencer area, so there is more room for the UI of the add-on. I hope you'll make your voice heard on the topic? :-)
